src/GitHub.Exports/Helpers/SimpleRepositoryModelExtensions.cs

```diff
@@ -0,0 +1,25 @@
+﻿using GitHub.Models;
+using System;
+using System.Linq;
+using System.IO;
+
+namespace GitHub.Extensions
+{
+    using VisualStudio;
+
+    public static class SimpleRepositoryModelExtensions
+    {
+        public static bool HasCommits(this ISimpleRepositoryModel repository)
+        {
+            var repo = Services.GetRepoFromPath(repository.LocalPath);
+            return repo?.Commits.Count() > 0;
+        }
+        public static bool HasContent(this ISimpleRepositoryModel repository)
+        {
+            var dir = new DirectoryInfo(repository.LocalPath);
+            return dir.EnumerateFileSystemInfos("*", SearchOption.TopDirectoryOnly)
+                      .Any(x => ((x.Attributes.HasFlag(FileAttributes.Directory) || x.Attributes.HasFlag(FileAttributes.Normal)) &&
+                                !x.Name.StartsWith(".", StringComparison.Ordinal) && !x.Name.StartsWith("readme", StringComparison.OrdinalIgnoreCase)));
```

Comment:
Ok, so it's a heuristic. Maybe a better name for the method is `MightContainSolution` or `ShouldAllowOpeningInVisualStudio` so the method name is scoped to the one specific use case. I worry that I might later see this method name and call it when I need to determine whether a repo has content (ok, I won't, but maybe someone else will).